src/main/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogTableGeneratorSpanner.java

```diff
@@ -45,6 +47,27 @@ public class CreateDatabaseChangeLogTableGeneratorSpanner
           + "    deployment_id string(MAX),\n"
           + ") primary key (id, author, filename);";
 
+  final String createPostgresqlTableSql =
+      ""
+          + "CREATE TABLE public.__DATABASECHANGELOG__\n"
+          + "(\n"
+          + "    id            varchar(255) not null,\n"
```

Comment:
Here also: Replace all `varchar(255)` columns with just `varchar` to keep it consistent with GoogleSQL.